webpack/assets/javascripts/react_app/components/common/EmptyState/EmptyStatePrimaryActionButton.js

```diff
@@ -23,14 +23,14 @@ const PrimaryActionButton = ({ action }) => {
 
 // eslint-disable-next-line react/prop-types
 const urlButton = ({ url, title }) => (
-  <Button href={url} bsStyle="primary" bsSize="large">
+  <Button component="a" href={url} variant="primary">
     {title}
   </Button>
 );
 
 // eslint-disable-next-line react/prop-types
 const onClickButton = ({ onClick, title }) => (
-  <Button onClick={onClick} bsStyle="primary" bsSize="large">
+  <Button component="a" onclick={onClick} variant="primary">
```

Comment:
Not sure if we tried that already,
but what if we just return the button with `onClick` and `href`, something like:
```js
<Button component="a" onclick={onClick} href={url} variant="primary">
```
`url` should defaults to `#` / `''` to redirect nowhere

Reply:
I was thinking more in the terms of:
```
if (action.url || action.onClick) {
  if (!action.url) {
    action.url = '';
  }
  return urlButton(action);
}

const urlButton = ({ url, onClick, title }) => (
  <Button component="a" href={url} onclick={onClick} variant="primary">
    {title}
  </Button>
);
```

What do you think?

Reply:
I think that we could implement it directly in `defaultEmptyState.js`
current usage:
```js
action={action ? <PrimaryActionButton action={action} /> : null}
```

we could change it into something like:
```js
action={action ? <Button component="a" onclick={action.onClick} href={action.url || ''} variant="primary"> : null}
```

it would be best to pass `action` to `EmptyStatePattern` and declare it there IMO